api/src/main/resources/api-ehcache.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<ehcache name="apiCache">
+
+    <diskStore path="user.home"/>
+
+    <defaultCache maxElementsInMemory="1000" eternal="false"
+                  timeToIdleSeconds="30" timeToLiveSeconds="30" overflowToDisk="false"
+                  diskPersistent="false" diskExpiryThreadIntervalSeconds="45"/>
+
+    <cache name="userSearchLocales"
+           maxElementsInMemory="5000"
+           eternal="true"
+           overflowToDisk="false"
+           timeToIdleSeconds="12000"
+           timeToLiveSeconds="12000"
+           diskPersistent="false"
+           diskExpiryThreadIntervalSeconds="120"
+           memoryStoreEvictionPolicy="LRU"
+    />
```

Comment:
Yes, you are right, default caches are created. 

> We could provide more granular adjustments of config for module developers in a separate issue if needed.

I think it is a good idea